nc-news-fe/src/components/ArticleContent.jsx

```diff
@@ -0,0 +1,13 @@
+const ArticleContent = ({currentArticle}) => {
+    return (
+        <>
+        <h1>{currentArticle.title}</h1>
+        <img src={currentArticle.article_img_url} alt="" />
+        <p>{currentArticle.body}</p>
+        <h4>{currentArticle.author}</h4>
+        <h4>{currentArticle.created_at}</h4>

```

Comment:
Consider a nicer way of presenting this date that is more human readable, day.js is a good library for that.